buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -1463,4 +1463,13 @@ final void setIndex0(int readerIndex, int writerIndex) {
     final void discardMarks() {
         markedReaderIndex = markedWriterIndex = 0;
     }
+
+    /**
+     * Set leakDetector's LeakListener.
+     *
+     * @param leakListener If leakListener is not null, it will be notified once a ByteBuf leak is detected.
+     */
+    public static void setLeakListener(ResourceLeakDetector.LeakListener leakListener) {
```

Comment:
I think this method should be in `ByteBufUtil`. Placing it here will make it appear on all `AbstractByteBuf` sub-classes, such as `CompositeByteBuf`.